src/main/webapp/resources/js/components/AnalysesQueue.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from "react";
+import { Tag, Tooltip } from "antd";
+import { fetchAnalysesQueueCounts } from "./../apis/analysis/analysis";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faRunning } from "@fortawesome/free-solid-svg-icons";
+import { SPACE_XS } from "../styles/spacing";
+import styled from "styled-components";
+import { formatNumber } from "../utilities/number-utilities";
+import { blue4 } from "../styles/colors";
+
+const LabelTD = styled.td`
+  font-weight: bold;
+`;
+
+const ValueTD = styled.td`
+  width: 50px;
+  text-align: right;
+  font-family: monospace;
+`;
+
+/**
+ * React component for rendering the current server status
+ * for running analyses.
+ * @return {*}
+ * @constructor
+ */
+export function AnalysesQueue({}) {
+  const [running, setRunning] = useState(0);
+  const [queued, setQueued] = useState(0);
+
+  useEffect(() => {
+    fetchAnalysesQueueCounts().then(data => {
+      setRunning(data.running);
+      setQueued(data.queued);
+    });
+  }, [fetchAnalysesQueueCounts]);
+
+  return (
+    <Tooltip title={i18n("AnalysesQueue.title")} placement={"bottomRight"}>
```

Comment:
Can we target this on either the left or bottom center? it gives me very weird overflows depending on screen size.

Reply:
Fixed